internal/resources/policy/kind/network/resource/resource_network_policy_test.go

```diff
@@ -281,35 +281,36 @@ func TestAcceptanceForNetworkPolicyResource(t *testing.T) {
 
 	t.Log("Network policy resource acceptance test complete for deny-all-egress recipe!")
 
+	// TODO: Skip custom-egress and custom-ingress tests currently for mock unitl the issue: OLYMP-40111 is fixed.
 	// Test case for network policy resource with custom-egress recipe.
 	resource.Test(t, resource.TestCase{
 		PreCheck:          testhelper.TestPreCheck(t),
 		ProviderFactories: testhelper.GetTestProviderFactories(testConfig.Provider),
 		CheckDestroy:      nil,
 		Steps: func() []resource.TestStep {
-			steps := []resource.TestStep{
-				{
-					Config: testConfig.getTestNetworkPolicyResourceBasicConfigValue(scope.WorkspaceScope, policykindNetwork.CustomEgressRecipe),
-					Check:  testConfig.checkNetworkPolicyResourceAttributes(scope.WorkspaceScope, policykindNetwork.CustomEgressRecipe),
-				},
-				{
-					PreConfig: func() {
-						if testConfig.ScopeHelperResources.OrgID == "" {
-							t.Skip("ORG_ID env var is not set for organization scoped network policy acceptance test")
-						}
-					},
-					Config: testConfig.getTestNetworkPolicyResourceBasicConfigValue(scope.OrganizationScope, policykindNetwork.CustomEgressRecipe),
-					Check:  testConfig.checkNetworkPolicyResourceAttributes(scope.OrganizationScope, policykindNetwork.CustomEgressRecipe),
-				},
-			}
-
 			if found {
-				steps = append(steps, resource.TestStep{
-					Config: testConfig.getTestNetworkPolicyResourceBasicConfigValue(scope.WorkspaceScope, policykindNetwork.CustomEgressRecipe, WithPodLabel("key3", "value3")),
-					Check:  testConfig.checkNetworkPolicyResourceAttributes(scope.WorkspaceScope, policykindNetwork.CustomEgressRecipe),
-				})
+				steps := []resource.TestStep{
+					{
+						Config: testConfig.getTestNetworkPolicyResourceBasicConfigValue(scope.WorkspaceScope, policykindNetwork.CustomEgressRecipe),
```

Comment:
nit: There is function called skipTest. Is it possible to make use of it instead?

Reply:
Skip is for individual tests , since here all tests had to be skipped I chose this route. This is temporary until we have fixed the tests so I think this should be good.